explain/setup/crunchy_bridge/_01_in_app.mdx

```diff
@@ -0,0 +1,34 @@
+import AutoExplainNotice from '../_auto_explain_notice.mdx'
+
+export const AutoExplainEnabled = () => {
+  return (
+    <React.Fragment>
+      <p>
+        It looks like <code>auto_explain</code> is already enabled on this server.
+      </p>
+      <Link className="btn btn-success" to="03_review_settings">
+        Skip to Step 3: Review auto_explain settings
+      </Link>
+    </React.Fragment>
+  )
+}
+
+export const AutoExplainNotEnabled = () => {
+  return (
+    <React.Fragment>
+      <p>
+        It looks like <code>auto_explain</code> is not enabled on this server.
+      </p>
+      <AutoExplainNotice />
+      <Link className="btn btn-success" to="02_enable_auto_explain">
+        Proceed to Step 2: Enable auto_explain
+      </Link>
+    </React.Fragment>
+  )
+}
+
+export const AutoExplainCheck = ({enabled}) => {
+  return enabled ? <AutoExplainEnabled /> : <AutoExplainNotEnabled />
+}
+
+<AutoExplainCheck enabled={props.autoExplainEnabled} />
```

Comment:
note: this doesn't return `true` when it's enabled with `session_preload_libraries` currently, though I'll make a PR to support it too in app, so let's assume that it's doing its job in this PR.